shell/platform/darwin/ios/framework/Source/FlutterViewController.mm

```diff
@@ -62,6 +67,18 @@ @interface FlutterViewController () <FlutterBinaryMessenger, UIScrollViewDelegat
 @property(nonatomic, assign) double targetViewInsetBottom;
 @property(nonatomic, retain) CADisplayLink* displayLink;
 
+/*
+ * Trackpad gesture recognizers
+ */
+@property(nonatomic, retain)
+    UIPanGestureRecognizer* discretePanGestureRecognizer API_AVAILABLE(ios(13.4));
```

Comment:
Now that we don't use scoped_nsobject, we need to manually release them.
Dealoc is probably the best place to do it.